src/renderer/services/wallet/keystore.ts

```diff
@@ -34,6 +35,26 @@ export const removeKeystore = async () => {
   setKeystoreState(O.none)
 }
 
+/**
+ * Exports a keystore
+ */
+const exportKeystore = async (runeNativeAddress: string, selectedNetwork: Network) => {
+  try {
+    const keystore: CryptoKeystore = await window.apiKeystore.get()
+    const defaultFileName =
+      selectedNetwork === 'testnet'
+        ? `asgardex-keystore-${runeNativeAddress.substring(0, 8)}_${runeNativeAddress.substring(
+            runeNativeAddress.length - 3
+          )}.json`
+        : `asgardex-keystore-${runeNativeAddress.substring(0, 7)}_${runeNativeAddress.substring(
+            runeNativeAddress.length - 3
+          )}.json`
```

Comment:
You can use [truncateAddress](https://github.com/thorchain/asgardex-electron/pull/748/files#diff-6e29a42bfa6dc3cea76cf566b1a31a271cece91bae9ec49e6006d28c8bf27ebcR3), which is available in master now